src/app/pages/edit-profile/edit-profile.page.ts

```diff
@@ -152,7 +158,11 @@ export class EditProfilePage implements OnInit, isDeactivatable {
         });
         this.form1.myForm.markAsPristine();
         const updated = await this.profileService.profileUpdate(form);
-        if(updated){ this.router.navigate([`${CommonRoutes.TABS}/${CommonRoutes.HOME}`], { replaceUrl: true })}
+        if(updated && this.redirectUrl){ 
```

Comment:
You are not using the redirect url for redirecting. Can u pass the url inside params? And pass that directly to router.navigate function